snowshu/templates/credentials_schema_target.json

```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Validation schema for credentials.yml",
+    "description": "credentials.yml store the target connection information.",
+    "type": "object",
+    "properties": {
+      "targets": {
+        "type": "array",
+        "items": [
+          {
+            "type": "object",
+            "properties": {
+              "account": {
+                "type": "string"
+              },
+              "adapter": {
+                "type": "string"
+              },
+              "database": {
+                "type": "string"
+              },
+              "name": {
+                "type": "string"
+              },
+              "password": {
+                "type": "string"
+              },
+              "user": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "account",
+              "adapter",
+              "database",
```

Comment:
We might not actually need `database` here since snowshu fully qualifies everything. We can always remove it later if unneeded.